KomaMRICore/src/simulation/Bloch/BlochSimulationMethod.jl

```diff
@@ -54,10 +59,11 @@ function run_spin_precession!(p::Phantom{T}, seq::DiscreteSequence{T}, sig::Abst
     tp = cumsum(seq.Δt) # t' = t - t0
     dur = sum(seq.Δt)   # Total length, used for signal relaxation
     Mxy = [M.xy M.xy .* exp.(1im .* ϕ .- tp' ./ p.T2)] #This assumes Δw and T2 are constant in time
+    M.z .= M.z .* exp.(-dur ./ p.T1) .+ p.ρ .* (1 .- exp.(-dur ./ p.T1))
     M.xy .= Mxy[:, end]
-    M.z  .= M.z .* exp.(-dur ./ p.T1) .+ p.ρ .* (1 .- exp.(-dur ./ p.T1))
     #Acquired signal
```

Comment:
No real reason to do this.

Reply:
Resolved in [665f6f6](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/665f6f6188ad0c2fc9b9511de3e70f91337b78a2)